tests/sentry/search/events/builder/test_metrics.py

```diff
@@ -21,10 +21,10 @@
 from sentry.sentry_metrics.use_case_id_registry import UseCaseID
 from sentry.sentry_metrics.utils import resolve_tag_value
 from sentry.snuba.dataset import Dataset
+from sentry.snuba.metrics.extraction import QUERY_HASH_KEY
 from sentry.testutils.cases import MetricsEnhancedPerformanceTestCase
 
-pytestmark = pytest.mark.sentry_metrics
-from sentry.snuba.metrics.metric_extraction import QUERY_HASH_KEY
+# pytestmark = pytest.mark.sentry_metrics
```

Comment:
do we need this comment here?

Reply:
No, I'll remove it if tests pass. I'm not sure what this line used to do, and for some reason tests in this module do not want to run locally for me.